src/instruments/src/SD/Pages/Fuel/Fuel.tsx

```diff
@@ -0,0 +1,266 @@
+            <>
+                {/* Main ducts */}
+                <line className="FlowShape" x1={160} y1={84} x2={160} y2={76} />
+
+                {/* Left engine LP valve */}
+                <EngineLpValve x={160} y={100} engineNumber={1} />
+
+                {/* Ducts to left inner pumps */}
+                <line className="FlowShape" x1={160} y1={215} x2={160} y2={116} />
+                <line className="FlowShape" x1={160} y1={190} x2={195} y2={190} />
+                <line className="FlowShape" x1={195} y1={190} x2={195} y2={215} />
+
+                {/* Pumps */}
+                <Pump x={145} y={215} active={!!pumpLeftInner1} />
+                <Pump x={180} y={215} onBus="DC_2" active={!!pumpLeftInner2} />
+
+                {/* Quantities */}
+                <text className="TankQuantity" x={74} y={285}>{Math.round(tankLeftOuter)}</text>
+                <text className="TankQuantity" x={190} y={285}>{Math.round(tankLeftInner)}</text>
+            </>
+
+            {/* Center */}
+            <>
+                {/* Side lines */}
+                <line className="FlowShape" x1="160" y1="150" x2="270" y2="150" />
+                <line className="FlowShape" x1="330" y1="150" x2="440" y2="150" />
+
+                {/* Center valve lines */}
+                {(crossFeedPosition === 100) && (
+                    <>
+                        <line className="FlowShape" x1="317" y1="150" x2="330" y2="150" />
+                        <line className="FlowShape" x1="270" y1="150" x2="283" y2="150" />
+                    </>
+                )}
+
+                {/* CrossFeed valve */}
+                <CrossFeedValve x={300} y={150} />
+
+                {/* Center pumps lines */}
+                <line className="FlowShape" x1="274" y1="150" x2="274" y2="210" />
+                <line className="FlowShape" x1="326" y1="150" x2="326" y2="210" />
+
+                {/* Pumps */}
+                <Pump x={259} y={205} active={!!pumpCenter1} />
+                <Pump x={311} y={205} active={!!pumpCenter2} />
+
+                {/* Quantities */}
+                <text className="TankQuantity" x={335} y={275}>{Math.round(tankCenter)}</text>
+            </>
+
+            {/* Right */}
+            <>
+                {/* Main duct */}
+                <line className="FlowShape" x1={440} y1={84} x2={440} y2={76} />
+
+                {/* Right engine LP valve */}
+                <EngineLpValve x={440} y={100} engineNumber={2} />
+
+                {/* Ducts to right inner pumps */}
+                <line className="FlowShape" x1={440} y1={215} x2={440} y2={116} />
+                <line className="FlowShape" x1={440} y1={190} x2={405} y2={190} />
+                <line className="FlowShape" x1={405} y1={190} x2={405} y2={240} />
+
+                {/* Pumps */}
+                <Pump x={390} y={215} active={!!pumpRightInner1} />
+                <Pump x={425} y={215} onBus="DC_2" active={!!pumpRightInner2} />
```

Comment:
By the way, when I wrote this I talked with @komp1821 or someone else at some point about the DC_2 bus thing, and I think he said it might not be the pump itself on the bus, but the overhead switch I think. Might wanna rename this prop then

Reply:
All Fuel pumps are AC powered, so I would say anything involving a DC bus is related to the control. I don't remember the exact conversation. If there's anything to confirm in terms of power sources, feel free to contact me. (Preferably on Discord)

Reply:
Isssue discussed and resolved with komp. Thanks.